ledger/blockqueue.go

```diff
@@ -319,7 +319,7 @@ func (bq *blockQueue) getBlockAux(r basics.Round) (blk bookkeeping.Block, aux ev
 		var err0 error
 		blk, aux, err0 = blockGetAux(tx, r)
 		return err0
-	})
+	}, r)
```

Comment:
Seems like a good test for this PR, but it would be cleaner to leave this code consistent: either all `blockQueue.getBlock*()` methods should pass this `r` as extras, or none should.